packages/flutter/test/material/scaffold_test.dart

```diff
@@ -147,6 +147,25 @@ void main() {
     expect(tester.binding.transientCallbackCount, greaterThan(0));
   });
 
+  testWidgets('Floating action button show/hide', (WidgetTester tester) async {
+    final GlobalKey one = new GlobalKey(debugLabel: 'one');
+    await tester.pumpWidget(new MaterialApp(home: new Scaffold(
+      floatingActionButton: new FloatingActionButton(
+        key: one,
+        onPressed: null,
+        child: const Text('1'),
+      ),
+    )));
```

Comment:
nit: add trailing commas to ensure that each property shows up on a separate line.

Reply:
This is copy pasted from the test above - I assumed it had been done this way for readability.  Should I change it?

Reply:
It's fine to leave it as is then.